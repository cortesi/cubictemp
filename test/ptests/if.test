@!"a" if 1 else "b"!@

@!"a" if 0 else "b"!@

<!--(block foo)
a
b
(end)-->
@!foo if 1 else "none"!@

@!foo if 0 else "none"!@

@!"safe" if 1 else 1/0!@
