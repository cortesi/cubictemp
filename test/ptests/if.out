a

b

a
b


none

safe
